input-output-hk/cardano-rosetta
# Feat/vasil hf

# Description

Support Vasil HF

# Proposed Solution

- [x] Bump Cardano Node and Cardano Db Sync versions.
- [x] Update test snapshot

# Testing

- Automated test suite
- Run automated scripts at `/examples`
- Run check:data with vasil-qa config file
- Run check:construction send transaction
